orchestra/frontend/build/asset-manifest.json

```diff
@@ -1,25 +1,25 @@
 {
```

Comment:
all changes in orchestra/frontend are rebuilds of ORCHESTRA_URL/orchestra/newapp after we removed `preventNew`